### DISCUSSION TITLE
Postgres PGVector

### DISCUSSION THREAD

**Discussion by mlvdheiden** [langflow-ai/langflow#4749] [General]
It is unclear to me what syntax I need to use for the connection URL. Is it postgresql+psycopg://[user]:[pwd]@[host]:[port]/[db] or what is the correct syntax? Looked in the documentation and did not found any hint

**Comment by mlvdheiden** [comment]
Sorry, found the answer myself :). 
But the integration demands a 'custom_id' column. This is not part of the original schema. Can I just add the column?
There's more now it needs a column 'uuid' also. Am I missing an update of the vector extension?

**Comment by rizalgunawan** [comment]
> It is unclear to me what syntax I need to use for the connection URL. Is it postgresql+psycopg://[user]:[pwd]@[host]:[port]/[db] or what is the correct syntax? Looked in the documentation and did not found any hint

can you tell me the correct syntax?

**Reply by luizhsalazar** [reply]
Correct syntax is postgresql+psycopg://[user]:[pwd]@[host]:[port]/[db], as @mlvdheiden mentioned in the original question.
